### PR TITLE
Add icons for CalyxOS

### PR DESCRIPTION
**Pull request by mwchiao**
## Description
Adding new icons for a few CalyxOS apps and linking existing icons to some apps

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* Work Profile (`org.calyxos.bellis`)
* Firewall (`org.calyxos.datura`)
* Backup (`com.stevesoltys.seedvault`)

### Icons linked
* Calendar (linked `org.lineageos.etar` to `@drawable/calendar`)
* Weather (linked `org.secuso.privacyfriendlyweather` to `@drawable/weather`)
* Recorder (linked `org.calyxos.recorder` to `@drawable/recorder`)

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -142,6 +143,7 @@
     <icon drawable="@drawable/calculator" package="com.oneplus.calculator" name="Calculator" />
     <icon drawable="@drawable/calculator" package="com.sec.android.app.popupcalculator" name="Calculator" />
     <icon drawable="@drawable/calculator_plus" package="com.gigaworks.tech.calculator" name="Calculator Plus" />
+    <icon drawable="@drawable/calendar" package="org.lineageos.etar" name="Calendar" />
```
Comment (daywalk3r666) [inline app/src/main/res/xml/grayscale_icon_map.xml:146]: https://github.com/LawnchairLauncher/lawnchair/commit/aa251677483fde500a0f34fd31e761bdb44566c7

**Review by Chefski — CHANGES_REQUESTED** [review]
Hey! Thanks for contributing! Got small changes:

`seedvault.svg` and `datura.svg` have strokes with an opacity that is not 100% Could you fix that?
image

Could you also remove the calendar link?

**Comment by mwchiao** [comment]
Changes made. One of these days I'll get everything right on the first try xD

Since the calendar link is being removed, is the only solution to make a new icon for the Etar calendar?

**Comment by daywalk3r666** [comment]
With the next release of lawnchair launcher it will be themed

**Review by Chefski — APPROVED** [review]
Thanks

**Pull request merged**